docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -50,7 +67,7 @@ The first `kubectl` command above installed the cass-operator and the Prometheus
 
 ### Create secret for read/write access to an S3 bucket
 
-Before creating the k8ssandra-cluster, we need to supply credentials so that Medusa has read/write to an S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and other bucket types. In this example, we’re using S3.
+Before creating the k8ssandra-cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and other bucket types. In this example, we’re using S3.
```

Comment:
It is probably worth noting that while Medusa supports  those, k8ssandra does not yet support local. I believe Medusa supports Azure as well, but again k8ssandra does not yet support it.

Reply:
Thanks, incorporating with next commit.